user/serializers.py

```diff
@@ -1 +1,30 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+from user.models import User
```

Comment:
The import of `User` from `user.models` is redundant since `get_user_model()` is used to retrieve the user model. You can remove this import to clean up the code.